src/backend/TrafficCourts/Citizen.Service/Models/Dispute/TicketDispute.cs

```diff
@@ -17,10 +18,13 @@ public class TicketDispute
         public string DriversLicence { get; set; }
         public string DriversLicenceProvince { get; set; }
         public string WorkPhone { get; set; }
-        [JsonConverter(typeof(DateOnlyJsonConverter))]
+
+        //[JsonConverter(typeof(DateOnlyJsonConverter))]
```

Comment:
Probably should have removed these commented lines, however, this is deprecated code.